central-pet-back/.env.development

```
@@ -2,7 +2,7 @@
 NODE_ENV=development
 PORT=3000
 FRONTEND_URL=http://localhost:5173
-DATABASE_URL=postgresql://centralpet:centralpet123@localhost:5432/centralpetdb
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/centralpetdb
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the FRONTEND_URL key

(UnorderedKey)